transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -872,12 +892,90 @@ private AbstractWriteTask(Recycler.Handle handle) {
             super(handle);
         }
 
+        private static int estimateSize(Object o, Map<Class<?>, Integer> classSizes) {
+            int answer = 8 + estimateSize(o.getClass(), classSizes, null);
+
+            if (o instanceof ByteBuf) {
```

Comment:
@Scottmitch yep and like I said I think we not need it.... at least not for now. I would like to not start over engineer stuff if not really needed. If at some point we find out we need it we can still change it. Sounds good ?